### ISSUE TITLE
fast-neural-style uses mscoco but normalizes for imagenet mean

### ISSUE THREAD

**Issue by twairball** [pytorch/examples#187]
Documentation for `fast-neural-style` uses mscoco training dataset, but subtracts imagenet mean from image input data. 

The effects are probably very minor, but anybody have the mean stats for mscoco?

**Comment by Kaixhin** [comment]
Are you talking about the use of `normalize_batch` within `neural_style.py`? Looks like this is applied before feeding images into a pretrained VGG net, which was trained on ImageNet. Whether or not normalisation is applied to images fed into the transformer network is a completely separate matter.
